open_bounties/0089 Xrpl-py Event Handler.md

```diff
@@ -0,0 +1,66 @@
+<!-- Please update this title -->
+
+# Xrpl-py Event Handler
+
+<!--
+Please include this line to ensure that no work is started prior to merging and budget approval
+-->
+
+Bounty proposals require 3+ technical approvals _and_ budget approval before they are available to be worked on. A proposal PR with 3+ ✅s is not fully approved if it has not been merged. The bounty is only ready to be worked on after the PR has been merged.
+
+**Category: Client Library Feature**
+
+<!--
+Recommend a total value amount for the bounty, in U.S. Dollars. The exact amount will be determined by the approving committee and may be higher or lower than this recommendation.
+-->
+
+**Amount:** $2,500
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+Xrpl.js currently allows you to subscribe to ledger updates ([as specified here](https://xrpl.org/subscribe.html)) - it'd be useful to allow that in xrpl-py as well.
```

Comment:
@mDuo13 Since you originally created the [inspiring issue here](https://www.github.com/XRPLF/xrpl-py/issues/290), I have a couple questions: 

1. Does this description make sense, or am I conflating two things which aren't the same in this description of the job to be done? 

2. What is the `.off` method supposed to do?

Reply:
`.off` should remove the subscription.